NEWS.md

```diff
@@ -22,6 +22,12 @@
   explicitly quoting key variables such as `STUDYID` and `USUBJID`. For instance, a line such as `by_vars = exprs(STUDYID, USUBJID)`
   would be replaced by `by_vars = get_admiral_option("subject_keys")` (#226).
 
+# admiralophtha 1.1.0
+
+## Updates to Templates
+
+- Within the ADBCVA template, updated `AVALCA1N` and `AVALCAT1` derivations to use new function `admiral::derive_vars_cat()`. (#244)
```

Comment:
@steventing12 this shouldn't be under admiralophtha 1.1.0 as that version has already released - it should be under 1.0.0.9007

Reply:
Updated, please check again